<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>680</width>
    <height>541</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Documasonry</string>
  </property>
  <widget class="QPushButton" name="load_infomation_files_button">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>-40</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;从文件里载入要填充的数据，&lt;/p&gt;&lt;p&gt;文件可以是 *.inf *.txt *.xls 格式&lt;/p&gt;&lt;p&gt;或者直接从资源管理器拖放文件到上面的区域&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="text">
    <string>载入数据...</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="templates_table">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>291</width>
     <height>191</height>
    </rect>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Sunken</enum>
   </property>
   <property name="alternatingRowColors">
    <bool>false</bool>
   </property>
   <attribute name="horizontalHeaderStretchLastSection">
    <bool>true</bool>
   </attribute>
   <attribute name="verticalHeaderStretchLastSection">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QPushButton" name="set_output_path_button">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>270</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>成果路径...</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="output_path_textedit">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>270</y>
     <width>571</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="info_textedit">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>30</y>
     <width>361</width>
     <height>231</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>宋体</family>
     <pointsize>11</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QPushButton" name="detect_required_fields_button">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>230</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;也可以从资源管理器拖放文件到左侧区域&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="text">
    <string>读取模板,检测其中字段</string>
   </property>
  </widget>
  <widget class="QPushButton" name="generate_button">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>330</y>
     <width>661</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>按照字段内容填充模板</string>
   </property>
  </widget>
  <widget class="QPushButton" name="invert_select_templates_button">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>230</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;也可以从资源管理器拖放文件到左侧区域&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="text">
    <string>反选</string>
   </property>
  </widget>
  <widget class="QPushButton" name="select_all_templates_button">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>230</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;也可以从资源管理器拖放文件到左侧区域&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="text">
    <string>全选</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="color_logger">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>390</y>
     <width>661</width>
     <height>141</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>201</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>模板列表(*.doc *.xls *.dwg ...)</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>10</y>
     <width>201</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>字段属性</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="add_index_checker">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>310</y>
     <width>131</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>生成文件添加序号</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
